crates/admin/src/cluster_controller/service.rs

```diff
@@ -740,18 +731,20 @@ impl SealAndExtendTask {
             #[cfg(any(test, feature = "memory-loglet"))]
             ProviderConfiguration::InMemory => (
                 ProviderKind::InMemory,
-                u64::from(loglet_id.next()).to_string().into(),
+                rand::random::<u64>().to_string().into(),
             ),
             ProviderConfiguration::Local => (
                 ProviderKind::Local,
-                u64::from(loglet_id.next()).to_string().into(),
+                rand::random::<u64>().to_string().into(),
```

Comment:
or even here. I think we can just use the loglet id (log_id, segment) everywhere.